Automattic/vip-go-mu-plugins
# Update ElasticPress submodule to latest on master branch

## Description

Update ElasticPress to latest from `master`

## Checklist

Please make sure the items below have been covered before requesting a review:

- n/a This change has relevant unit tests (if applicable).
- n/a This change has relevant documentation additions / updates (if applicable).

## Steps to Test


1. Check out PR.
1. `git submodule update --init --recursive`
1. Test various search functionality
